touch-layouts/layouts/red-dead-redemption-2.json

```diff
@@ -0,0 +1,272 @@
+{
```

Comment:
Use this tool to format json (indent = 2): https://duckduckgo.com/?t=ffab&q=format+json&ia=answer